### ISSUE TITLE
Error Message: WordPress database error You have an error in your SQL syntax

### ISSUE THREAD

**Issue by alecmuffett** [akirk/friends#268]
in the logfiles; I am guessing that `relationships_post_format.term_taxonomy_id` is not meant to be empty?

This is a fresh WP6.4 instance set up to handle ActivityPub and nothing else, with all plugins up to date, set up last night.

```
2023/11/07 21:52:35 [error] 118890#118890: *410767 FastCGI sent in stderr:
"PHP message: WordPress database error You have an error in your SQL syntax;
check the manual that corresponds to your MariaDB server version for the right syntax to use near ')
AND relationships_author.object_id = posts.ID
AND taxonomy_auth...' at line 10 for query SELECT relationships_post_format.term_taxonomy_id AS post_format_id, COUNT(relationships_post_format.term_taxonomy_id) AS count
FROM ewwp_posts AS posts
JOIN ewwp_term_relationships AS relationships_post_format
JOIN ewwp_term_taxonomy AS taxonomy_author
JOIN ewwp_term_relationships AS relationships_author
WHERE posts.post_status IN ( 'publish', 'private' )
AND posts.post_type IN ( 'friend_post_cache' )
AND relationships_post_format.object_id = posts.ID
AND relationships_post_format.term_taxonomy_id IN (  )
AND relationships_author.object_id = posts.ID
AND taxonomy_author.term_taxonomy_id = relationships_author.term_taxonomy_id
AND taxonomy_author.term_id = 26" while reading upstream...
```

**Comment by akirk** [comment]
Thank you for reporting this. It occurs when the theme doesn't have post format support and you haven't activated settings to provide post format support regardless of the theme's support.

This should be fixed with #271.

**Comment by alecmuffett** [comment]
Awesome, thank you Alex; I've ticked the box now and hope that will improve
matters until the fix ships.
